block-storage/src/test/scala/coop/rchain/blockstorage/dag/BlockDagStorageTest.scala

```diff
@@ -31,6 +31,8 @@ trait BlockDagStorageTest
   "DAG Storage" should "be able to lookup a stored block" in {
     forAll(blockElementsWithParentsGen(genesis), minSize(0), sizeRange(10)) { blockElements =>
       withDagStorage { dagStorage =>
+        implicit val bds = dagStorage
+        implicit val s   = Sync[Task]
```

Comment:
With changes to DagRepresentation methods not returning F values this can be avoided.